doc/building.md

```diff
@@ -55,6 +55,17 @@ cmake ..
 make -j$(nproc)
 ```
 
+_Note:_ on windows is
+```powershell
+cd nyan
+mkdir build
+cd build
+cmake ..
+cmake --build . --config RelWithDebInfo
```

Comment:
ok, now the commands are basically identical apart from this line -> we can just unify and say "if you used makefiles as generator target (linux, mac, ...): `make ...`
if you are on windows or want to let cmake start the build: `cmake --build ...`"

then its not redundant but still acurate :)

Reply:
ok fix